src/chainparams.cpp

```diff
@@ -156,6 +156,7 @@ class CMainParams : public CChainParams {
         vSeeds.push_back(CDNSSeedData("dashjr.org", "dnsseed.bitcoin.dashjr.org"));
         vSeeds.push_back(CDNSSeedData("bitcoinstats.com", "seed.bitcoinstats.com"));
         vSeeds.push_back(CDNSSeedData("xf2.org", "bitseed.xf2.org"));
+        vSeeds.push_back(CDNSSeedData("bitcoin.jonasschnelli.ch", "seed.bitcoin.jonasschnelli.ch", true));
```

Comment:
I added your seed to my DNS seed monitoring (http://bitcoinstats.com/network/dns-servers/) looks nice and stable, I assume you take the XORed peer IPs out of the same pool you use to serve the non-XORed, right?